test/utils/mocks/body.html

```diff
@@ -39,6 +39,28 @@
     <p>
       <a class="copy-action" href="https://www.adobe.com/#_evt-copy"></a>
     </p>
+    <!-- Aria labels appendment -->
+    <p>
+      <a href="https://www.adobe.com/" class="aria-label-none">Text</a>
+    </p>
+    <p>
+      <a href="https://www.adobe.com/" class="aria-label-simple">Text | Aria label</a>
+    </p>
+    <p>
+      <a href="https://www.adobe.com/" class="aria-label-piped">Text | Other text | Aria label</a>
```

Comment:
We should also test use-cases where there's no space between words separated by `|`.

Reply:
Added another unit test 👍